src/utils/index.ts

```diff
@@ -0,0 +1 @@
+export * from './notifications.js';
```

Comment:
I think we want to avoid those `index.ts`, so you can export the file directly in the root.